dev-db/sqlitestudio/sqlitestudio-3.4.3.ebuild

```diff
@@ -110,13 +128,23 @@ src_prepare() {
 src_configure() {
 	# NOTE: QMAKE_CFLAGS_ISYSTEM option prevents
 	# build error with tcl use enabled (stdlib.h is missing)
+	# "QMAKE_CFLAGS_ISYSTEM=\"\""
+	# CONFIG+ borrowed from compile.sh of tarball
 	local myqmakeargs=(
 		"BINDIR=${EPREFIX}/usr/bin"
 		"LIBDIR=${EPREFIX}/usr/$(get_libdir)"
-		"QMAKE_CFLAGS_ISYSTEM=\"\""
+		"CONFIG+=portable"
 		$(usex test 'DEFINES+=tests' '')
 	)
 
+	# Combination of kvirc ebuild and qtcompress
+	if use python; then
+		myqmakeargs+=(
+			INCLUDEPATH+="$(python_get_includedir)"
+			LIBS+="$(python_get_library_path)"
```

Comment:
This does not work (results in a link error).  You want `$(python_get_LIBS)` instead.

Also I usually prepend a space in this context, in case LIBS already contained something.  (Same for INCLUDEPATH above)